controllers/snapshot/snapshot_adapter_test.go

```diff
@@ -1081,25 +981,23 @@ var _ = Describe("Snapshot Adapter", Ordered, func() {
 					ContextKey: loader.SnapshotContextKey,
 					Resource:   hasInvalidSnapshot,
 				},
-				{
-					ContextKey: loader.EnvironmentContextKey,
-					Resource:   env,
-				},
 				{
 					ContextKey: loader.SnapshotComponentsContextKey,
 					Resource:   []applicationapiv1alpha1.Component{*hasComp},
 				},
 				{
 					ContextKey: loader.AllIntegrationTestScenariosContextKey,
-					Resource:   []v1beta1.IntegrationTestScenario{*integrationTestScenarioForInvalidSnapshot},
+					Resource:   []v1beta2.IntegrationTestScenario{*integrationTestScenarioForInvalidSnapshot},
 				},
 				{
 					ContextKey: loader.RequiredIntegrationTestScenariosContextKey,
-					Resource:   []v1beta1.IntegrationTestScenario{*integrationTestScenarioForInvalidSnapshot},
+					Resource:   []v1beta2.IntegrationTestScenario{*integrationTestScenarioForInvalidSnapshot},
 				},
 			})
-			result, err := adapter.EnsureStaticIntegrationPipelineRunsExist()
+
+			result, err := adapter.EnsureIntegrationPipelineRunsExist()
 			Expect(result.CancelRequest).To(BeFalse())
+			// TODO: revert this to BeTrue
```

Comment:
This line will be updated once the https://github.com/redhat-appstudio/integration-service/pull/572/
will see light of day.

Reply:
Its set to false in main, ignore previous comment.